3416.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- BSD-3 Clause License
 
-  Copyright 2023 uCIFI Alliance
+  Copyright 2020-2025 uCIFI Alliance
```

Comment:
@gmazzoleni, @mkgillmore The changes in this object 3416 are substantial; these changes are not editorial. The Object version should be updated to reflect these changes; however, the DDF does not indicate a new version. Please check.